doc/src/Howtorun.tex

```diff
@@ -106,4 +106,20 @@ \section{TEAL for RAVEN}
 which can range from 0 to 100, 0 meaning maximum debug verbosity and 100 meaning
 errors only. Setting the verbosity to 50 will output (in addition to errors) the
  NPV, IRR, PI or NPV\_mult. Inside the \xmlNode{Economics} block, there are two
- types of blocks: \xmlNode{Global} and \xmlNode{Component}.
\ No newline at end of file
+ types of blocks: \xmlNode{Global} and \xmlNode{Component}.
+
+An example of a cash flow is shown in Listing \ref{lst:CashFlowExample}. In the example, a cash flow called CAPEX is defined.
+In the example, the capital expenditure for a reference plant of capacity
+1'000'000'000 W is \$4 billion. The driver of this cash flow is the actual plant capacity in Watts. Building this plat has some economy
```

Comment:
plat -> plant

Reply:
I know there are a few other typos in here. I'll find and fix the rest when I do the rewrite.